packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -167,6 +167,9 @@ export const getPolymarketMarketInformation = async (
   if (!market) {
     throw new Error(`No market found for question ID: ${questionID}`);
   }
+  if (!market.clobTokenIds) {
+    throw new Error(`Market found for question ID: ${questionID} has no clobTokenIds`);
```

Comment:
we see 2 different errors, one related to no market found ( error above) the other to not having any clobTokenIds.  This will now check that explicitly and make sure we throw and error in this case

Reply:
I could be interesting to log the market and see what's inside, maybe it's empty or maybe just missing the `clobTokenIds`? wdyt?

Reply:
i thought about doing that, would i just do a logger.info call? and do i need to explicitly pass the logger in?

Reply:
added it